### PR TITLE
UX improvement to banner warnings and errors

### PR DESCRIPTION
**Pull request by kenzieschmoll**
This changes the message on the performance page and cpu profiler page from an "error" to a "warning", and it tweaks the wording of this message. This PR also changes the message style a little bit by using an error/warning icon instead of the words "ERROR" and "WARNING".
Screen Shot 2021-12-13 at 3 56 39 PM
(note: this error screenshot is just to show the styling, and the text here is actually shown as a warning - not an error)
Screen Shot 2021-12-13 at 4 04 44 PM

@InMatrix @cobblest

### CODE REVIEW

File: packages/devtools_app/lib/src/banner_messages.dart [hunk 1]
```diff
@@ -184,10 +185,65 @@ class BannerMessage extends StatelessWidget {
...
+
+  WidgetSpan _iconSpanForMessage() {
+    Widget child;
+    switch (messageType) {
+      case BannerMessageType.warning:
+        child = const _BannerWarningIcon();
+        break;
+      case BannerMessageType.error:
+      default:
+        child = Icon(
```
[the remark this answers is not in the document]
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/banner_messages.dart]: done

**Comment by InMatrix** [comment]
Thanks for turning this into a PR. At the moment, there is a bit of a gap between what we tell users what they can do in debug mode and what they would find easy to do. In addition to UI improvements we might make to the perf page, I'm hoping to turn each task mentioned in the warning message (e.g., "building widgets") into a link to a how-to guide.

**Comment by kenzieschmoll** [comment]
> I'm hoping to turn each task mentioned in the warning message (e.g., "building widgets") into a link to a how-to guide.

We may be able to do this fairly soon. I know @johnpryan has been looking into writing some documentation for these features.

**Comment by kenzieschmoll** [comment]
We will hold off on landing this PR until we have more actionable information we can point users to (flutter.dev documentation, debugging guides, etc.).

**Comment by Hixie** [comment]
Is there an issue # for creating those docs?

**Comment by kenzieschmoll** [comment]
@InMatrix are you opposed to landing this?

**Pull request merged**